helm-grafana/manifests/preflight.yaml

```diff
@@ -44,15 +44,6 @@ spec:
           - pass:
               when: "== aks"
               message: AKS is a supported distribution
-          # Will be supported in the future
-          - pass:
-              when: "== kurl"
-              message: KURL is a supported distribution
-          - pass:
-              when: "== digitalocean"
-              message: DigitalOcean is a supported distribution
-          - warn:
-              message: Unable to determine the distribution of Kubernetes
```

Comment:
did you want to remove the else clause here?

Reply:
Oh, I see. Yeah, I just wanted to remove what wasn't supported.